src/ops/op.c

```diff
@@ -240,6 +247,26 @@ void op_poke_i16(const void *data, scene_state_t *ss, exec_state_t *NOTUSED(es),
     tele_vars_updated();
 }
 
+void op_peek_seed_i16(const void *data, scene_state_t *ss,
+                      exec_state_t *NOTUSED(es), command_state_t *cs) {
+    char *base = (char *)ss;
+    size_t offset = (size_t)data;
+    rand_set_t *ptr = (rand_set_t *)(base + offset);
```

Comment:
does this depend on specific order of members in `scene_state_t`?

Reply:
no I don't think order matters.  I'm using the same technique as the `MAKE_SIMPLE_VARIABLE_OP` macro, but the warning comment for variable order is related to something else.

https://llllllll.co/t/teletype-firmware-discussion/13961/38?u=alphacactus